sdk/postgresql/Azure.ResourceManager.PostgreSql/src/autorest.md

```diff
@@ -192,8 +192,6 @@ rename-mapping:
   FailoverMode: PostgreSqlFlexibleServerFailoverMode
   FlexibleServerEditionCapability: PostgreSqlFlexibleServerEditionCapability
   GeoRedundantBackupEnum: PostgreSqlFlexibleServerGeoRedundantBackupEnum
-  HyperscaleNodeEditionCapability: PostgreSqlFlexibleServerHyperscaleNodeEditionCapability
-  NodeTypeCapability: PostgreSqlFlexibleServerNodeTypeCapability
```

Comment:
Why we removed some configurations? If they still work, we should not remove them because this causes breaking changes.

Reply:
These properties are no longer supported and hence they were removed. They belong to a separate capability which was clubbed with Flexible server. We no longer support this offering in capability call for flexible servers and hence this was removed